.github/workflows/pr.yaml

```diff
@@ -54,4 +54,4 @@ jobs:
         run: make
       - name: Test documentation builds
         if: matrix.os == 'ubuntu-latest'
-        run: make documentation
+        run: make documentation
```

Comment:
@vrathi101 was this supposed to be changed?

Reply:
> @vrathi101 was this supposed to be changed?

no, I was trying to revert it but I don't know how cuz there's no extra empty line in my version of the file

Reply:
we should have an extra empty line